### DISCUSSION TITLE
Edit the Navbar message using the INVENTREE_CUSTOMIZE Variable?

### DISCUSSION THREAD

**Discussion by E-Hain** [inventree/InvenTree#6895] [Q&A]
What is the proper way to edit the Navbar message using the INVENTREE_CUSTOMIZE Variable? I have tried using it and it throws an error in the Inventree logs saying that it needs to be a "INVENTREE_CUSTOMIZE must be a valid JSON string". Can I get some help with how this is supposed to be done?

**Comment by SchrodingersGat** [comment]
Can you provide an example of the data you are trying to provide?

**Reply by E-Hain** [reply]
Yeah so for instance... just as a simple test. How would I display a "Hello world" with data provided from the .env variable? My end goal would be to provide a link eventually
